src/templates/react-native/scripts/fastlane.sh

```diff
@@ -1,28 +1,14 @@
 #!/bin/sh
 
 echo '
-###
 1. Installing `graphicsmagick` 
-fastlane-plugin-app-icon dependency
-###'
+A `fastlane-plugin-app-icon` dependency'
 brew install graphicsmagick
 
-# Enter ios folder
-cd ./ios
-
 echo '
-###
-2. iOS: Installing Fastlane
-###'
+2. Installing Fastlane'
 bundle install
 
 echo '
-###
-3. iOS: Running Fastlane:beta
-###'
-bundle exec fastlane beta
-
-cho '
-###
-4. Android: Not implement yet
-###'
\ No newline at end of file
+3. Running Fastlane:beta'
```

Comment:
```suggestion
3. Running Fastlane'
```
Not really. Actually, the Fastlane decides if it's a beta or a production build while it's running the pre-build.